.github/workflows/pkgci_regression_test.yml

```diff
@@ -254,10 +260,49 @@ jobs:
             --log-cli-level=info \
             --timeout=1200 \
             --durations=0 \
-            --config-files=${SDXL_CONFIG_FILE_PATH}
+            --config-files=${SDXL_UNET_CONFIG_FILE_PATH}
+
+      - name: "Run external tests - SDXL prompt encoder"
+        if: "matrix.sdxl-clip-config-file != '' && !cancelled()"
```

Comment:
Let's continue keeping an eye on complexity here. New models shouldn't necessarily need workflow file changes.

1. Add imported model to the external test suite with .mlir, .irpa, and input/output .bin files
2. Bump test suite commit hash in this repo, choose which models from the test suite to run (on which backends, with which flags, etc.)

For these models I have the feeling that pytest (or at least this approach to pytest) is the wrong tool for the job. It's a tool that we have though, so fine for now...